components/eamxx/docs/user/model_output.md

```diff
@@ -89,6 +98,12 @@ I/O interface of EAMxx. There are two types of diagnostic outputs:
     - `precip_ice_surf_mass_flux`
     - `precip_total_surf_mass_flux`
     - `surface_upward_latent_heat_flux`
+    - `wind_speed`
+    - `AerosolOpticalDepth550nm`
+    - `NumberPath`
+    - `AeroComCld`
+
+  TODO: add some information about what each diagnostic is, perhaps a formula
```

Comment:
We should make a git-issue for this todo.

Reply:
Yeah, We could probably just put the formula in the header files, and then just say "for details on the formula for each diagnostic, see the comment at the top of their header file in `components/eamxx/src/diagnostics`.